server/src/test/java/org/elasticsearch/discovery/ClusterDisruptionIT.java

```diff
@@ -111,7 +140,9 @@ public void testAckedIndexing() throws Exception {
         final AtomicReference<CountDownLatch> countDownLatchRef = new AtomicReference<>();
         final List<Exception> exceptedExceptions = new CopyOnWriteArrayList<>();
 
-        logger.info("starting indexers");
+//        final ConflictMode conflictMode = ConflictMode.randomMode();
```

Comment:
I think it's ok to chose this randomly instead of having three separate tests. Especially as this test typically takes a bit of time to run.

Reply:
I wanted my PR builds to have all 3 variants running. Will change to randomMode before merging to master.